.github/workflows/update_llm_perf_intel_pytorch.yml

```diff
@@ -0,0 +1,57 @@
+name: Update LLM Perf Benchmarks - CUDA PyTorch
+
+on:
+  pull_request:
+  push:
```

Comment:
avoid this

Reply:
yes, i know i am just doing some debugging at the moment

Reply:
it triggers the workflow with the incomplete PR, which pushes to the llm perf dataset

Reply:
Ok thanks for the info, right now it will fails before that. I was just testing if the intel runner was picking up the job